pkg/api/defaults-kubelet.go

```diff
@@ -187,7 +187,7 @@ func removeKubeletFlags(k map[string]string, v string) {
 	}
 
 	// Get rid of values not supported in v1.15 and up
-	if common.IsKubernetesVersionGe(v, "1.15.0-beta.1") {
+	if common.IsKubernetesVersionGe(v, "1.15.0-alpha.3") {
```

Comment:
Why this change?

Reply:
@jackfrancis Was using that for testing with custom hyperkube until 1.15 beta support was added. Removed it now.